<?xml version="1.0"?>
<launch>  
  <arg name="chain_start" default="link1" />
  <arg name="chain_end" default="end_effector_link" />

  <param name="robot_description" command="$(find xacro)/xacro.py '$(find open_manipulator_with_tb3_description)/urdf/open_manipulator_with_tb3_waffle_pi.urdf.xacro'" />

  <node name="trac_ik_teleop" pkg="open_manipulator_with_tb3_tools" type="trac_ik_teleop" output="screen">    
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="urdf_param" value="/om_with_tb3/robot_description"/>
  </node>
</launch>
